tests-e2e/cypress/integration/channels/post_type_components_spec.js

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('channels > post type components', () => {
+    let testTeam;
+    let testUser;
+    let testChannel;
+    let testPlaybookRun;
+
+    beforeEach(() => {
+        cy.apiInitSetup({promoteNewUserAsAdmin: true}).then(({team, user}) => {
```

Comment:
Re:
![CleanShot 2022-10-13 at 16 40 31@2x](https://user-images.githubusercontent.com/11724372/195716273-410e1334-b640-49fc-a75e-7e923b968476.png)
Maybe the following?
```suggestion
        cy.apiInitSetup({loginAfter: true, promoteNewUserAsAdmin: true}).then(({team, user}) => {
```

Reply:
Seems to have done it

Reply:
Huh. Thank you!!!